addons/l10n_pl_jpk/models/res_company.py

```diff
@@ -0,0 +1,9 @@
+from odoo import fields, models
+
+
+class Company(models.Model):
+    _inherit = 'res.company'
+
+    l10n_pl_reports_tax_office_id = fields.Many2one('l10n_pl_tax_office', string='Tax Office')
+    partner_is_company = fields.Boolean(related="partner_id.is_company")
+    birthdate = fields.Date(string='Birth Date')
```

Comment:
How would you save it as a config parameter, if we have several companies.?
Would you save it suffixed by the id of the company? (It feels hacky to me, and seems more like something we do because we don't have the choice in stable). It would be lighter on the db, that's for sure

Reply:
You're right ; it's hacky ... 
But indeed, adding a field just for that is ... :/ If there was some line with this info on the report, we could make it a manual value, but I guess it's not an option. Or at least, this would be a major change in the way we handle report exports, and would require changes elsewhere, for homogeneity.

So, should we just let the field on the wizard, without default value ? To me, it'd be acceptable (but not ideal at all), but the PO probably won't like this.

Reply:
The PO agreed to leave it empty without default value.
At worst, if people complain too much, we can always do the hacky solution at that point (it can be done in stable, so why not :smile: ).